configen.xcodeproj/project.pbxproj

```diff
@@ -284,6 +284,7 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
```

Comment:
Had to increase the deployment target from 10.10 (Yosemite, 2014) to 10.11 (El Capitan) as `URL(fileURLWithPath: String, relativeTo: URL)` is only available from 10.11.